reflex/components/libs/chakra.py

```diff
@@ -253,3 +253,28 @@ class ChakraColorModeProvider(Component):
 LiteralPopOverTrigger = Literal["click", "hover"]
 
 LiteralHeadingSize = Literal["lg", "md", "sm", "xs", "xl", "2xl", "3xl", "4xl"]
+
+LiteralChakraInputType = Literal[
```

Comment:
is this really chakra-specific or are these valid for regular html input type too?

Reply:
hmm, yeah thats a good point. Should work with regular html input as well. Will change that